superset/assets/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -64,49 +65,79 @@ const propTypes = {
 
 const defaultProps = {
   directPathToChild: [],
+  directPathLastUpdated: 0,
 };
 
 class ChartHolder extends React.Component {
-  static renderInFocusCSS(labelName) {
+  static renderInFocusCSS(columnName) {
     return (
       <style>
-        {`.inFocus label[for=${labelName}] + .Select .Select-control {
-                    border: 2px solid #00736a;
+        {`label[for=${columnName}] + .Select .Select-control {
```

Comment:
Why is it again that we need to add inline style elements to the react element?

Reply:
because filter_box component is deeply nested inside Chart component, also it is a viz type, its props needs to be validated from superset-ui chart props. it's 100 time more troublesome than inline css.

Reply:
seems legit